BossMod/BossModule/BossModule.cs

```diff
@@ -306,7 +306,15 @@ private void DrawPartyMembers(int pcSlot, Actor pc)
         foreach (var (slot, player) in Raid.WithSlot().Exclude(pcSlot))
         {
             var (prio, color) = CalculateHighestPriority(pcSlot, pc, slot, player);
-            if (prio == BossComponent.PlayerPriority.Irrelevant && !WindowConfig.ShowIrrelevantPlayers)
+
+            bool isFocus = false;
+            if (Service.TargetManager.FocusTarget != null)
```

Comment:
This will break UIDev, and in general replays. Most of the classes in vbm should not access game state directly.

Reply:
I'm not super familiar with this codebase, would it be more appropriate change the behavior so that it checks if a party member is focus-targeted in `BossMod/Framework/WorldStateGameSync.cs` around `UpdateParty`? Something like storing the slot of the focus'd player?

Reply:
Yeah, that's probably the right way - you'll need to add the focustarget to the world state (more precisely, i guess clientstate, since it's really player-specific) and check that here

Reply:
I've added focus target to client state now.

Reply:
Thanks for getting to this, I was a bit busy the past couple days. I went ahead and adjusted it to use `WorldState.Client.FocusTargetId`. Let me know if I did this incorrectly.